bazel/buildifier/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@buildifier_prebuilt//:rules.bzl", "buildifier", "buildifier_test")
+
+exclude_patterns = [
+    "**/.bazelbsp/**",
+    "**/.cache/**",
+]
+
+buildifier(
+    name = "buildifier",
+    exclude_patterns = exclude_patterns,
+    lint_mode = "fix",
+    mode = "fix",
+)
+
+# PROS of `buildifier_test` over `buildifier`:
+# - renders diffs without having to provide `tkdiff`
+# - also works as a `test` action: `bazel test //bazel/buildifier:test` as well as `bazel test //...`
+# CONS:
+# - requires additional arguments: `no_sandbox` and `workspace`
```

Comment:
boooo

Reply:
Right but, well, `run` actions are anyway not sandboxed by default...